backend/src/main/java/hu/flowacademy/worksheet/repository/WorksheetRepository.java

```diff
@@ -1,9 +1,13 @@
 package hu.flowacademy.worksheet.repository;
 
 import hu.flowacademy.worksheet.entity.Worksheet;
+import hu.flowacademy.worksheet.enumCustom.WorksheetStatus;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface WorksheetRepository extends JpaRepository<Worksheet, String> {
+    List<Worksheet> findByWorksheetStatusLike(WorksheetStatus status);
```

Comment:
Lehetne a szűrés SQL In-nel megoldva és akkor egyszerre több státusz is szűrhető. Hasznos, ha mondjuk mindenre kíváncsiak vagyunk.  Azt hiszem, tudja a JPA kezelni ha variadic paramként adod meg.